hiku/expr/compiler.py

```diff
@@ -71,10 +71,18 @@ def visit(self, node):
             return self.generic_visit(node)
 
     def generic_visit(self, node):
-        if isinstance(node, int):
+        if node is None:
+            return py.NameConstant(node)
+        elif isinstance(node, bool):
+            return py.NameConstant(node)
+        elif isinstance(node, integer_types):
+            return py.Num(node)
+        elif isinstance(node, float):
             return py.Num(node)
         elif isinstance(node, text_type):
             return py.Str(node)
+        elif isinstance(node, tuple):
```

Comment:
Only scalar types should be handled here, all the container types should be recursively transformed into expressions in the `hiku.expr.core._to_expr` function.

In the Hiku's expressions AST tree `Tuple` node is used to define function calls, as it does in Lisp-like languages. In the future this probably will be changed, as there are no benefits in copying Lisp/Clojure AST and introducing such disambiguation for the users.